src/main/java/org/example/Kingdom/Main.java

```diff
@@ -132,15 +120,7 @@ else if (kingType == 4) {
         else {
             game.beforeDay1_1();
         }
-        for (int i=0; i<5;) {
-            try {
-                Thread.sleep(250);
-            } catch (InterruptedException ex) {
-                Thread.currentThread().interrupt();
-            }
-            System.out.print(".");
-            i++;
-        }
+        sleep250_5times();
```

Comment:
```suggestion
        sleep(250, 5);
```

리뷰 남겨드린것과 같은 형태로 할 수 있습니다.